src/domUpdates.js

```diff
@@ -258,12 +258,14 @@ const createIngredientsHTML = ingredients => {
 const searchForRecipes = () => {
   const data = {
     ourRecipes: recipeData,
-    yourRecipes: null
+    yourRecipes: currentUser.recipesToCook
   }
   let searchedRecipes = searchRecipes(data[pageData.currentView], ingredientsData, searchBar.value)
-  searchedRecipes.length  
+  if(searchedRecipes) {
+    searchedRecipes.length  
     ? renderGrid(searchedRecipes)
```

Comment:
eventually, we'll just upgrade the data model first (recipesOfInterest) and then render from there right?

Reply:
Yes! In order to have it work now we need this, but that will get changed

Reply:
Awesome